packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss

```diff
@@ -2,15 +2,18 @@
 
 .root {
   @include focus-border();
-  flex-shrink: 0;
+  flex-shrink: 1;
   background: $ms-color-neutralLighter;
   height: 32px;
   line-height: 32px;
   cursor: default;
   user-select: none;
   display: flex;
   flex-wrap: nowrap;
+  min-width: 100px;
   max-width: 100%;
+  margin: 2px 2px;
```

Comment:
I'm doing some input height consistency work that includes the pickers, so if we don't need to tweak margin/padding here, let's avoid it. I want this PR to focus on the overflow issues only. @gitjain please just revert the margin/padding to what we have in master.